src/main/osd/osd_elements.c

```diff
@@ -1052,9 +1054,9 @@ static void osdElementMotorDiagnostics(osdElementParms_t *element)
         if (motorsRunning) {
             element->buff[i] =  0x88 - scaleRange(motor[i], getMotorOutputLow(), getMotorOutputHigh(), 0, 8);
 #if defined(USE_ESC_SENSOR) || defined(USE_DSHOT_TELEMETRY)
-            if ( getEscRpm(i) < 700 ) {
-                // motor is not spinning properly. Mark as Stopped
-                element->buff[i] =  0x53; // S
+            if (getEscRpm(i) < MOTOR_STOPPED_THRESHOLD_RPM) {
+                // Motor is not spinning properly. Mark as Stopped
+                element->buff[i] =  'S';
```

Comment:
```suggestion
                element->buff[i] = 'S';
```

Reply:
remove extra space

Reply:
besides that lgtm

Reply:
Thanks, done.

Also rebased and squashed commits